src/server/ocpp/services/OCPPService.ts

```diff
@@ -633,8 +639,14 @@ export default class OCPPService {
       await OCPPUtils.priceTransaction(headers.tenantID, transaction, consumption, TransactionAction.START);
       // Billing
       await OCPPUtils.billTransaction(headers.tenantID, transaction, TransactionAction.START);
+      // Is it possible that a TagID is registered at Gireve and Hubject? Then only one Session for either Gireve or Hubject should be started.
```

Comment:
Yes, it's possible. But for let's assume for now that a tenant can only have one roaming provider setup at a time. So you will only call the right method by checking the tenant settings. Do that for all call to roaming provider methods inside the transactions handling code.

Reply:
done